browser/misc_metrics/extension_metrics.cc

```diff
@@ -0,0 +1,82 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/misc_metrics/extension_metrics.h"
+
+#include "base/containers/fixed_flat_set.h"
+#include "base/metrics/histogram_macros.h"
+#include "extensions/browser/extension_registry.h"
+
+namespace misc_metrics {
+
+namespace {
+
+constexpr auto kPopularAdBlockExtensions =
+    base::MakeFixedFlatSet<std::string_view>({
+        // uBO
+        "cjpalhdlnbpafiamejdnhcphjbkeiagm",
+        // Ghostery
+        "mlomiejdfkolichcflejclcbmpeaniij",
+        // AdGuard
+        "bgnkhhnnamicmpeenaelnjfhikgbkllg",
+        // AdBlocker Ultimate
+        "ohahllgiabjaoigichmmfljhkcfikeof",
+    });
+constexpr base::TimeDelta kReportDebounceTime = base::Seconds(10);
```

Comment:
why do we need a timer, if we capture install/uninstall events anyway?

Reply:
it's a workaround for this issue https://github.com/brave/brave-browser/issues/31481

sometimes the registry will dispatch an event twice, rapidly